src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1526,6 +1527,12 @@ public String toVariableNameString(EnchantmentOffer eo) {
 				.name("Transform Reason")
 				.description("Represents a transform reason of an <a href='events.html#entity transform'>entity transform event</a>.")
 				.since("2.8.0"));
+
+		Classes.registerClass(new EnumClassInfo<>(ChangeReason.class,  "experiencechangereason", "experience change reasons")
+			.user("(experience|[e]xp) ?change (reason|cause)s?")
```

Comment:
should the first question mark be there? just making sure since idk what it does

Reply:
Honestly not sure, I just guessed